backend/aml/base_settings.py

```diff
@@ -195,4 +195,4 @@
     }
 }
 
-SUBPATH = os.getenv('AML_SUBPATH', None)
+SUBPATH = os.getenv('AML_SUBPATH', '')
```

Comment:
Now that you are using `BASE_URL` and not `AML_SUBPATH` to prefix the image urls, is this change still needed?

Reply:
No, but I like the empty string fallback better, anyway. I twill still evaluate to `False` in the backend.